packages/core-components/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -130,8 +130,6 @@ export type Props = {
   subheader?: ReactNode;
   divider?: boolean;
   deepLink?: BottomLinkProps;
-  /** @deprecated Use errorBoundaryProps instead */
-  slackChannel?: string;
```

Comment:
We don't want to remove this just yet, same reason as the other ones. We're looking to clean up usages of deprecated symbols, not the deprecated symbols themselves.